pkgs/top-level/all-packages.nix

```diff
@@ -5709,6 +5709,8 @@ with pkgs;
 
   hyprland-protocols = callPackage ../applications/window-managers/hyprwm/hyprland-protocols { };
 
+  hyprlang = callPackage ../by-name/hy/hyprlang/package.nix { };
+
```

Comment:
```suggestion
```
remove this -- by-name doesn't need anything in the top-level/all-packages.nix

Reply:
Right, forgot.